manager/client/main.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	rpc_license "github.com/enfabrica/enkit/manager/rpc"
+	grpcCodes "google.golang.org/grpc/codes"
```

Comment:
why rename them? (usually renaming is only done if there is a naming conflicts in the import, but don't see any here)

Reply:
I chose to rename it to be consistent with the `grpcStatus` package I renamed since I categorized all these packages as part of the grpc library. Not required, but just my preference.